src/entry/logEntry.ts

```diff
@@ -22,7 +22,21 @@ export type LogEntryRequestData = {
   from: number;
   count: number;
   files: string[];
+
   sortingOrderDESC: boolean | undefined;
+
+  filters: {
+    date: RangeDate | undefined;
+    ip: string | undefined;
+    text: string | undefined;
+    regex: string | undefined;
+    classification: string | undefined;
+  };
+};
```

Comment:
make this into a "Filter" type

Reply:
should also have boolean "isRegex" and one field to pass either the text or regex